Python/setup.py

```diff
@@ -39,31 +39,50 @@
 '''
 
 import os
+from pathlib import Path
+import re
 import setuptools
 
+
+def read_requirements():
+    requirements = []
+    with open('./requirements.txt', 'r') as file:
+        for line in file:
+            # exclude comments
+            if re.match('^\\s*#.*', line):
+                # print('****************LINE--', line)
+                continue
+            requirements.append(str(line.lstrip().rstrip()))
+    return requirements
+
+
 dawgie = os.path.join ('dawgie', '__init__.py')
-deps = ['bokeh>=1.2',
-        'boto3>=1.7.80',
-        'cryptography>=2.1.4',
-        'dawgie-pydot3==1.0.10',
-        'GitPython>=2.1.11',
-        'matplotlib>=2.1.1',
-        'psycopg2>=2.7.4',
-        'pyparsing>=2.2',
-        'python-gnupg==0.4.4',
-        'pyxb==1.2.6',
-        'requests>=2.20.0',
-        'transitions==0.6.8',
-        'twisted>=18.7.0',
-        ]
 version = os.environ.get ('DAWGIE_VERSION', '0.0.0')
 with open (os.path.join (os.path.dirname (__file__), dawgie)) as f: t = f.read()
 t = t.replace ("'0.0.0'", "'{0}'".format (version))
 with open (os.path.join (os.path.dirname (__file__), dawgie), 'tw') as f:\
      f.write (t)
-with open (os.path.join (os.path.dirname (__file__), 'README.md'), 'rt') as f:\
-     description = f.read()
 
+data_files_locations = []
+
+read_me_file_dir_string = "."
+read_me_file = Path(__file__).absolute().parent / "README.md"
+if read_me_file.exists() is False:
+    read_me_file_dir_string = ".."
+    read_me_file = read_me_file.parent.parent / read_me_file.name
+data_files_locations.append((".", [read_me_file_dir_string + "/" + read_me_file.name]))
+
+license_file_dir_string = "."
+license_file = Path(__file__).absolute().parent / "LICENSE.txt"
+if license_file.exists() is False:
+    license_file_dir_string = ".."
+    license_file = license_file.parent.parent / license_file.name
+data_files_locations.append((".", [license_file_dir_string + "/" + license_file.name]))
+
+with read_me_file.open() as f:
+    description = f.read()
```

Comment:
One line if shorter than 80 characters

Reply:
[PEP 8 is a little unclear here.](https://www.python.org/dev/peps/pep-0008/#other-recommendations) My IDE advises against this.  
"Compound statements (multiple statements on the same line) are generally discouraged."

Reply:
Please see updated code.